integr8ly/installation
# 3scale operator

Install 3scale via the 3scale operator https://github.com/integr8ly/3scale-operator

**Verification**

* Run the install playbook and verify everything comes up as expected
* Verify all expected users are created in 3scale
* Verify expected identity provider is created in 3scale
* Verify you can login to 3scale via SSO

**Related PRs:**

* https://github.com/integr8ly/3scale-operator/pull/1
